client/src/document/index.tsx

```diff
@@ -170,6 +173,10 @@ export function Document(props /* TODO: define a TS interface for this */) {
   return (
     <>
       <ArticleActionsContainer doc={doc} />
+
+      {/* only include this if we are not server-side rendering */}
+      {!isServer && <OfflineStatusBar />}
```

Comment:
Wonder if this is still necessary if we assume `isOnline = true` without a `window` (see below).

```suggestion
      {<OfflineStatusBar />}
```

Reply:
Nope, SSR still fails.